package.json

```diff
@@ -23,6 +23,7 @@
     "test:all:cov": "jest --coverage --config ./test/jest-all.json"
   },
   "dependencies": {
+    "@gelatonetwork/relay-sdk": "^5.5.5",
```

Comment:
We should consider not using the `@gelatonetwork/relay-sdk`. As it stands it does not have tests – https://github.com/gelatodigital/relay-sdk/tree/v5.5.5 and also pulls yet another network library (`axios`)

Can be done in a follow up PR.

Reply:
I'll investigate this in a follow up.